scylla/templates/scylla-statefulset.yaml

```diff
@@ -0,0 +1,64 @@
+apiVersion: apps/v1beta1
+kind: StatefulSet
+metadata:
+  name:  {{ template "scylla.fullname" . }}
+  labels:
+    app: {{ template "scylla.fullname" . }}
+    chart: "{{ .Chart.Name }}-{{ .Chart.Version }}"
+    release: "{{ .Release.Name }}"
+    heritage: "{{ .Release.Service }}"
+spec:
+  serviceName: {{ template "scylla.fullname" . }}
+  replicas: {{ .Values.replicaCount }}
+  selector:
+    matchLabels:
+      app: {{ template "scylla.fullname" . }}
+  template:
+    metadata:
+      labels:
+        app: {{ template "scylla.fullname" . }}
+    spec:
+      containers:
+      - name: {{ template "scylla.fullname" . }}
+        image:  "{{ .Values.Image.repository }}:{{ .Values.Image.tag }}"
+        imagePullPolicy: {{ default "" .Values.Image.pullPolicy | quote }}
+        ports:
+          - containerPort: {{ .Values.Statefulset.intraPort }}
+            name: {{ .Values.Statefulset.intraName }}
+          - containerPort: {{ .Values.Statefulset.tlsIntraPort }}
+            name: {{ .Values.Statefulset.tlsIntraName }}
+          - containerPort: {{ .Values.Statefulset.jmxPort }}
+            name: {{ .Values.Statefulset.jmxName }}
+          - containerPort: {{ .Values.Service.port }}
+            name: {{ .Values.Statefulset.cqlName }}
+        lifecycle:
+          preStop:
+            exec:
+              command: ["/bin/sh", "-c", "PID=$(pidof scylla) && kill $PID && while ps -p $PID > /dev/null; do sleep 1; done"]
+        env:
+          - name: POD_IP
+            valueFrom:
+              fieldRef:
+                fieldPath: status.podIP
+        readinessProbe:
+          exec:
+            command:
+              - /bin/bash
+              - -c
+              - exec
+              - /opt/ready-probe.sh
+          initialDelaySeconds: 15
+          timeoutSeconds: 5
+        volumeMounts:
+          - name: scylla-data
+            mountPath: /var/lib/scylla
+          - name: scylla-ready-probe
+            mountPath: /opt/ready-probe.sh
+            subPath: ready-probe.sh
+      volumes:
+        - name: scylla-data # don't forget to change to persistent volume
+          emptyDir:
```

Comment:
This means that db is going to memory and will be discarded in a pod restart.

This may be enough for you now, but I would add a `TODO` here to add more flexibility to this.